test/run.sh

```diff
@@ -65,9 +91,8 @@ for dockerImage in "$@"; do
 		continue
 	fi
 	
-	noNamespace="${dockerImage##*/}"
```

Comment:
I suppose it would be possible to check for test configs for both the namespaced and the non-namespaced verson of the test, but this thing is pretty complicated already.

Also, since it's not documented, I wonder if anyone's ever taken advantage of this functionality.